src/RevPrefetcher.cc

```diff
@@ -12,9 +12,10 @@
 
 bool RevPrefetcher::IsAvail(uint64_t Addr){
 
-  // note: this logic does not consider compressed instructions
-  //       however, it should work for what we need.  we only need
-  //       to know whether an instruction as been filled
+  // note: this logic now considers compressed instructions
+  if( Addr == 0x4bd38ull ){
```

Comment:
remove address check

Reply:
resolved in https://github.com/tactcomplabs/rev/commit/e9a5df3d3de81136677d389638c82e5ced6b65c9